app/validators/order_compatibility_validator.rb

```diff
@@ -15,9 +15,9 @@ def validate(record) # rubocop:todo Metrics/CyclomaticComplexity
     return if order_request_types.all?(&:not_for_multiplexing?)
 
     record.errors.add(:orders, 'are incompatible') if order_request_types.any?(&:not_for_multiplexing?)
-    unless order_request_types.all? do |request_types|
+    unless order_request_types.all? { |request_types| # stree-ignore # rubocop:disable Style/BlockDelimiters
```

Comment:
OK now it's just messing with you.  Changing this from `do end` to `{ }` is not right.